functions/palindrome_number.py

```diff
@@ -1,2 +1,12 @@
 def palindrome_number(x):
-    pass
+  temp = x
+  reverse = 0
+  while(x>0):
+    dig = x%10
```

Comment:
dig could be more descriptive
```suggestion
    digit = x%10
```